README.md

```diff
@@ -1,3 +1,3 @@
-# **RND2020**
+# **RnD-2020**
 ## **Learning**
```

Comment:
```
# - is equivalent to h1
## - is equivalent to h2
```
I need you to find something which is equivalent to ul, ol and li in md. Please make the changes for line no `3` after you find those.

Reply:
should learning also be a list?

Reply:
Nope. Learning should be like side heading for the list below. From now, you'll be adding things to list items periodically based on the lessons and assignments done.

Reply:
bro,
I found equivalent from this link https://www.markdownguide.org/basic-syntax/ .What changes should I make in line 3?

Reply:
Sorry, my bad I thought there was another `*` at the end of line
* something like this *

It's okay if you use `*` or `-` for list item. 

Will merge this now.

Reply:
No problem bro.Thanks